lib/learn_test/ide/client.rb

```diff
@@ -0,0 +1,19 @@
+require 'socket'
+
+module LearnTest::Ide
+  IDE_USER_HOME = "/home/#{ENV['USER']}"
+
+  class Client
+    def browser_open(url)
+      File.open("#{ide_user_home}/.fs_changes.log", 'a') do |f|
+        f.puts "chrome BROWSER_OPEN #{url}"
```

Comment:
rather than this three column approach, let's just write a json string with a command key, ie

```ruby
f.puts { command: 'browser_open', url: url }.to_json
```